base/experimental.jl

```diff
@@ -494,4 +494,13 @@ function entrypoint(@nospecialize(argt::Type))
     nothing
 end
 
+"""
+    Base.Experimental.disable_new_worlds()
+
+Mark that no new worlds (methods additions, deletions, etc) are permitted to be
+created, allowing for greater performance and slightly lower memory usage by
```

Comment:
Performance here can be elaborated a bit on here I think. Presumably your for loops won't run faster. This has mostly to do with package loading?

Reply:
```suggestion
created, allowing for lower latencies and slightly lower memory usage by
```

Reply:
Okay, but when do you want to use this? This should have a description of a use case I feel.